cove_ofds/process.py

```diff
@@ -158,23 +161,51 @@ def process(self, process_data: dict) -> dict:
         if os.path.exists(self.data_filename):
             return process_data
 
-        output_dir = os.path.join(
-            self.supplied_data.data_dir(), CONVERT_CSVS_INTO_JSON_DIR_NAME
-        )
-
-        os.makedirs(output_dir, exist_ok=True)
-
-        schema = OFDSSchema
+        with tempfile.TemporaryDirectory() as tmp_dir_name:
+            print(tmp_dir_name)
```

Comment:
I'm assuming these 2 prints should be removed before merge?

Reply:
I had totally messed up my git branches, sorry, try again